web/pdf_page_view.js

```diff
@@ -563,23 +566,9 @@ class PDFPageView {
     }
     div.removeAttribute("data-loaded");
 
-    if (annotationLayerNode) {
-      // Hide the annotation layer until all elements are resized
-      // so they are not displayed on the already resized page.
-      this.annotationLayer.hide();
-    }
-    if (annotationEditorLayerNode) {
-      this.annotationEditorLayer.hide();
-    }
-    if (xfaLayerNode) {
-      // Hide the XFA layer until all elements are resized
-      // so they are not displayed on the already resized page.
-      this.xfaLayer.hide();
-    }
```

Comment:
Given that the xfaLayer is a bit special, is keeping it visible actually correct and safe in general?

*Ninja-edit:* Testing the preview the answer unfortunately seems to be NO, since scrolling to the end of an XFA document and then rotating it triggers intermittent rendering glitches (since pages that are not visible will not be updated until they are scrolled into view again).

Here's a screen-shot of what this could look like: 
![xfa](https://github.com/user-attachments/assets/4ecbb9e7-61c7-421d-89e9-e9edb96eeab7)